web/client/codechecker_client/cmd/cmd.py

```diff
@@ -1255,6 +1255,21 @@ def __register_del(parser):
     __add_common_arguments(del_t, needs_product_url=False)
 
 
+def __register_labels(parser):
```

Comment:
If I print the help message of this command it will have a --url option:
```common arguments:
  --url PRODUCT_URL     The URL of the product which will be accessed by the
                        client, in the format of
                        '[http[s]://]host:port/Endpoint'. (default:
                        localhost:8001/Default)
```
Why do we need this? For now I think we don't use it, so it can be removed. Later if a subcommand will require it, we will introduce it.